Source/ACE.Server/WorldObjects/WorldObject_Networking.cs

```diff
@@ -1068,7 +1068,10 @@ public virtual ACE.Entity.ObjDesc CalculateObjDesc()
         protected void AddBaseModelData(ACE.Entity.ObjDesc objDesc)
         {
             // Hair/head
-            if (HeadObjectDID.HasValue && !HairStyle.HasValue)
+
+            // if (HeadObjectDID.HasValue && !HairStyle.HasValue)
+            // This Heritage check has been added for backwards compatibility. It works around the butthead Gear Knights appearance.
+            if (HeadObjectDID.HasValue && !HairStyle.HasValue && Heritage.HasValue && Heritage != (int)HeritageGroup.Gearknight)
```

Comment:
I'm not sure if this is okay, or if we should do a check much earlier on just once and essentially kill this property there (set it to null). That was my initial thinking, but I couldn't see WHERE would be a good place for a check like that.